src/components/molecules/CountBySeverityPieWidget.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { FunctionComponent } from 'react'
+import PieChartView from './PieChartView'
+import { IWidgetCountBySeverity } from '../../models/WidgetData'
+
+interface IProps {
+	data: IWidgetCountBySeverity[]
+}
+const CountBySeverityPieWidget: FunctionComponent<IProps> = props => {
+	const x = 'accident_severity'
```

Comment:
These are constants. They never change.
Move them to the top of the file (under import) and change their name to CAPITAL_SNAKE_CASE.
Also, please give them clearer names than X and Y.